test/functional/vimscript/writefile_spec.lua

```diff
@@ -148,9 +148,9 @@ describe('writefile()', function()
       'Vim(call):E118: Too many arguments for function: writefile',
       pcall_err(command, ('call writefile([], "%s", "b", 1)'):format(fname))
     )
-    for _, arg in ipairs({ '0', '0.0', 'function("tr")', '{}', '"test"' }) do
+    for _, arg in ipairs({ '0', '0.0', 'function("tr")', '{}' }) do
       eq(
-        'Vim(call):E475: Invalid argument: writefile() first argument must be a List or a Blob',
+        'Vim(call):E475: Invalid argument: writefile() first argument must be a List, a Blob or String',
```

Comment:
This message should not be changed. In Lua a blob *is* a string. And in Vimscript, string is not accepted for this file.

Reply:
In this PR's implementation, a string is accepted. Otherwise it'll require
> information about whether each string comes from Lua to be available in typval strings.